__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+beforeEach(() => seed(data));
+
+afterAll(() => db.end());
+
+describe("/api/invalid-end-points", () => {
+  test("GET:404 responds with a 404 when passed an invalid path", () => {
+    return request(app).get("/api/nonsensefkjashslgh").expect(404);
```

Comment:
If we're having this 404 test in here, it's always going to pass at the moment due to the fact that Express has a default 404 error handler which will pick this up. If we're going to keep the test, could we please add some custom error handling which sends a custom message we can test for? 🙂